templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -177,14 +177,14 @@ objects:
         level: debug
         color: true
       openshift:
-        host: ${CLUSTER_AUTH_HOST}
-        ca_file: ${CA_FILE}
-        bearer_token_file: ${BEARER_TOKEN_FILE}
-        image_pull_policy: ${IMAGE_PULL_POLICY}
+        host: "${CLUSTER_AUTH_HOST}"
+        ca_file: "${CA_FILE}"
+        bearer_token_file: "${BEARER_TOKEN_FILE}"
+        image_pull_policy: "${IMAGE_PULL_POLICY}"
       broker:
         dev_broker: ${DEV_BROKER}
         bootstrap_on_startup: ${BOOTSTRAP_ON_STARTUP}
-        refresh_interval: ${REFRESH_INTERVAL}
+        refresh_interval: "${REFRESH_INTERVAL}"
         launch_apb_on_bind: ${LAUNCH_APB_ON_BIND}
```

Comment:
Yes booleans fail if you convert them into strings when umarshalling the yaml file. So everything is treated as a string other than bools.